word_frequency.py

```diff
@@ -1,13 +1,64 @@
+
+
+
+
 STOP_WORDS = [
     'a', 'an', 'and', 'are', 'as', 'at', 'be', 'by', 'for', 'from', 'has', 'he',
     'i', 'in', 'is', 'it', 'its', 'of', 'on', 'that', 'the', 'to', 'were',
     'will', 'with'
 ]
 
+punc_list = r'''!()-[]{};:'"\, <>./?@#$%^&*_~'''
+
 
 def print_word_freq(file):
-    """Read in `file` and print out the frequency of words in that file."""
-    pass
+    counts_dict = {}
+    
+    opened_file = open(file, 'r')
+    read_file = opened_file.read().lower()
+    
+    for elem in read_file:
+        if elem in punc_list:
+            read_file = read_file.replace(elem, " ")
+
+    read_file = read_file.split()
+
+    read_file_copy = read_file.copy()
+
+    for word in read_file:
+        if word in STOP_WORDS:
+            read_file_copy.remove(word)
+    
+    read_file_copy.sort()
+
+    for word in read_file_copy:
+        counts_dict[word] = read_file_copy.count(word)
+    
+    
+    # sorted_words = sorted(counts_dict.items(), key=lambda x:x[1], reverse=True)
+    
+    # print(counts_dict.items())
+
+    # print(counts_dict.keys())
+    # values_sorted = sorted(counts_dict.values(), key=None, reverse=True)
+    # print(values_sorted)
+
+    # for item in counts_dict:
+    #     key = counts_dict.keys()
```

Comment:
The dict.keys(), values(), & items() work on the whole dict. Right here, it looks like you are trying to apply these methods to one key, value pair in the dict.